dot/rpc/modules/childstate.go

```diff
@@ -68,3 +75,23 @@ func (cs *ChildStateModule) GetKeys(_ *http.Request, req *GetKeysRequest, res *[
 	*res = hexKeys
 	return nil
 }
+
+// GetStorageSize returns the size of a child storage entry.
+func (cs *ChildStateModule) GetStorageSize(_ *http.Request, req *GetChildStorageRequest, res *uint64) error {
+	if req.Hash == common.EmptyHash {
+		req.Hash = cs.blockAPI.BestBlockHash()
+	}
+
+	stateRoot, err := cs.storageAPI.GetStateRootFromBlock(&req.Hash)
+	if err != nil {
+		return err
+	}
+
+	item, err := cs.storageAPI.GetStorageFromChild(stateRoot, req.KeyChild, req.EntryKey)
+	if err != nil {
+		return err
+	}
+
+	*res = uint64(len(item))
```

Comment:
I think the return value here needs to be an optional, so if the `item == nil` then it returns `None` otherwise it returns `Some(len(item))`. currently if the item is nil or if the item has length=0 they'll both return 0

Reply:
@EclesioMeloJunior does this need to change?

Reply:
@timwu20 updated!